05.User-resource.md

```diff
@@ -114,6 +114,27 @@ Try it from the CLI:
 
 ### Unread Items
 
+You can retrieve unread items using the following endpoint.
+
+```
+GET /v1/user/:userId/rooms/:roomId/unreadItems
+```
+
+```
+{"chat":["560ae5f495756f1402bc841d","560aeef6f4b61c106fb2a9f3","560aef2b552ed7913279df52","560aef4f081f3a9c044d7d61","560aef66081f3a9c044d7d67","560b167a967c1bad7852bc57"],"mention":[]}
```

Comment:
Add one item to the `mention` array. Reduce `chat` items to 4. Please format this to be pretty.